### ISSUE TITLE
Preview function for seeing how the work done will look in a given front-end

### ISSUE THREAD

**Issue by lottebelice** [linkedtv/editortool#8]

**Comment by lyndonnixon** [comment]
I assume this is tied to the WP3 UI work too: i.e. the editor tool could show the same enrichment presentation as the LinkedCulture or LinkedNews applications e.g. the "information card" for an object.

**Comment by michielhildebrand** [comment]
Indeed. This relates to the functionality we needed to edit the slides for
Linked News. In the prototype editor tool we explored this
http://linkedtv.project.cwi.nl/newseditor/?source=src/snowden.json.
- you can add the 8 entities shown per chapter
- you can update how the entity is shown in the "thumbnail bar" in the
  second screen application: the thumbnail and label of the entity
- you can edit the (wiki) text shown about the entity in the slide. It
  would be nice to have metadata field here as well.
- you can also change when the entity slide is shown

It would be awesome is some of this functionality can be integrated in the
editor tool. The code is on
https://github.com/michielhildebrand/linkedtv_news_editor

I expect that for Linked Culture we will need some additional functionality.

Michiel
